adapter/config/types.go

```diff
@@ -552,6 +553,15 @@ type controlPlane struct {
 	InitialFetch               initialFetch
 }
 
+type asbDataplaneTopic struct {
```

Comment:
shall we include new configs to the template file?

Reply:
added via https://github.com/wso2/product-microgateway/pull/3604/commits/b3e3181bf85539da5e0b5c3a57ef7406baf5fe3f